torch_tvm/custom_tvm_ops/topi/quantize.cc

```diff
@@ -0,0 +1,89 @@
+#pragma once
```

Comment:
No `pragma once` here.

Reply:
Good catch! I guess I copied it from .h file.